packages/curve-ui-kit/src/lib/model/query/factory.ts

```diff
@@ -65,7 +54,8 @@ export function queryFactory<
     })
 
   return {
-    assertValidity,
+    assertValidity: (data: TParams, fields?: TField[]) =>
```

Comment:
A query's `assertValidity` is now literally never used. Only in that broken ass gauge reward feature that was barely completed is `assertValidity` used, and i nthat case it's the global version from `import { assertValidity } from '@ui-kit/lib/validation'` where you pass the validation suite as the parameter

Imo we can remove this exported prop entirely.